internal/kafka/internal/api/public/api/openapi.yaml

```diff
@@ -1701,16 +1701,24 @@ components:
         href: href
       properties:
         reason:
+          description: Human-readable description of the error. Intended for human
+            consumption
           type: string
         operation_id:
+          description: Relatively unique operation ID of the request associated to
+            the error
           type: string
         id:
+          description: The unique and immutable identifier of the resource
           type: string
         kind:
+          description: The kind of the resource
           type: string
         href:
+          description: The absolute path of the resource
           type: string
         code:
+          description: Code of the error
```

Comment:
This is the field that ideally should be used to perform programmatic error handling. However I intentionally haven't been that explicit in the description as we currently do not have a robust definition/handling of error codes